openpype/hosts/maya/plugins/load/load_vdb_to_redshift.py

```diff
@@ -87,3 +95,70 @@ def load(self, context, name=None, namespace=None, data=None):
             nodes=nodes,
             context=context,
             loader=self.__class__.__name__)
+
+    def _apply_settings(self,
+                        grid_node,
+                        path):
+        """Apply the settings for the VDB path to the VRayVolumeGrid"""
+        from maya import cmds
+
+        # The path points to a single file. However the vdb files could be
+        # either just that single file or a sequence in a folder so we check
+        # whether it's a sequence
+        folder = os.path.dirname(path)
+        files = os.listdir(folder)
+        is_single_file = len(files) == 1
+        if is_single_file:
+            filename = path
+        else:
+            # The path points to the publish .vdb sequence filepath so we
+            # find the first file in there that ends with .vdb
+            files = sorted(files)
+            first = next((x for x in files if x.endswith(".vdb")), None)
+            if first is None:
+                raise RuntimeError("Couldn't find first .vdb file of "
+                                   "sequence in: %s" % path)
+            filename = os.path.join(path, first)
```

Comment:
I think this should fix the issue?

```suggestion
            filename = os.path.join(folder, first)
```